content/projects/gsoc/mentors.adoc

```diff
@@ -128,6 +129,40 @@ Google publishes the list of accepted organizations according to the
 link:https://summerofcode.withgoogle.com/how-it-works/#timeline[Official GSoC timeline].
 If we are accepted, we move on to the next phase.
 
+[[forming_mentor_teams]]
+=== Mentors and mentor teams
+
+*Expectation:* a few hours before the <<ranking_and_slot_request_phase>>.
+
+We want mentors to form mentor teams of at least 2 or 3 mentors and to co-mentor projects together.
+Please spend a few hours forming your mentor team as soon as possible but no later than the start of the <<ranking_and_slot_request_phase>>.
+
+Do spend a few hours looking for co-mentors (mailing list, chat room, social media, etc.) and contact them.
+You should also setup introduction audio or video conference with them, and you should invite them to the office hours meeting.
+It is good to discuss your mutual interests (why you are interested in the same projects, your respective backgrounds in open source, etc.)
+and your respective availabilities during the program.
+
+Mentors must make sure they get invited by org admins, and must make sure to respond to that invitation
+by the time the <<ranking_and_slot_request_phase>> phase begins. It is possible for org admins to add mentors later, but that would be only
+if an already registered mentor could no longer participate.
+
+In the Jenkins organization, we have the concepts of lead mentor and co-mentors.
+
+The lead mentor has the responsibilities of a co-mentor, and is also primarily responsible for the general administrative duties
+commonly found in GSoC:
+
+* fill in the evaluation documents,
+* ensuring representation at office hours meetings and special meetings,
+* report issues to the Org Admin team.
+
+All mentors work with the student during all phases on the program,
+answering the student questions, coaching, advising, motivating, unblocking, reviewing code and pull-requests, ensuring the process is followed,
+ensuring communications are in public as this is an open source program, and report issues to the lead mentor, etc.
+
+==== Subject matter experts and community members
```

Comment:
Should be https://jenkins.io/projects/gsoc/roles-and-responsibilities/ , right now section does not look to be complete